### DISCUSSION TITLE
Add `Plain` option in Send Mail Flows options

### DISCUSSION THREAD

**Discussion by raflymln** [directus/directus#16656] [Feature Requests]
Hi Directus Team!

So even though I made a PR a long time ago about adding the WYSIWYG editor to the Send Mail Flows option, which I thought was enough, but now I don't think so, I think we need to create a `Plain` option, which allows us to send unparsed email content, because either the WYSIWYG or Markdown editor parses the content.

If my explanation is quite hard to understand, let me give you the example here:

1. Here i am trying to create an email template, using TailwindCSS, but in order for this css working, i need to specify Tailwind CDN script: https://tailwindcss.com/docs/installation/play-cdn

```html
<div class="mx-auto flex w-full max-w-screen-sm flex-col overflow-hidden text-black">
    <section class="relative flex flex-col gap-10 bg-yellow-light p-10">
        <img src="xxx" alt="logo" class="w-24" />

        <div class="z-10 flex flex-col gap-3">
            <p class="font-poppins">Hi <b>user@email.com</b>,</p>

            <h1 class="font-vidaloka text-5xl leading-snug">Welcome to The Subscription!</h1>
        </div>

        <img src="xxxx" alt="" class="absolute bottom-0 right-0 hidden h-full translate-x-1/4 transform sm:block" />
    </section>
</div>
```

2. I am trying to add `<script src="https://cdn.tailwindcss.com"></script>` into the content using the `Edit Source Code` feature on the WYSIWYG Editor

```html 
...
</div>
<script src="https://cdn.tailwindcss.com"></script>
```

3. After i save it and click the `Edit Source Code` button again, i get this:

```html
<div class="mx-auto flex w-full max-w-screen-sm flex-col overflow-hidden text-black">
<section class="relative flex flex-col gap-10 bg-yellow-light p-10"><img class="w-24" src="xxxx" alt="logo" />
<div class="z-10 flex flex-col gap-3">
<p class="font-poppins">Hi <strong>user@email.com</strong>,</p>
<h1 class="font-vidaloka text-5xl leading-snug">Welcome to The Subscription!</h1>
</div>
<img class="absolute bottom-0 right-0 hidden h-full translate-x-1/4 transform sm:block" src="xxx" alt="" /></section>
</div>
```

Okay so you see the content is being parsed by the editor, so i do some temporary fix by converting those class into styles, now i get this:

```html
<div style="margin-left: auto; margin-right: auto; display: flex; width: 100%; max-width: 640px; flex-direction: column; overflow: hidden; color: #272727;">
    <section style="position: relative; display: flex; flex-direction: column; gap: 2.5rem; background-color: #efd09e; padding: 2.5rem;">
        <img style="width: 6rem;" src="https://runes.asia/assets/42842041-0178-4253-9bb8-e53b05424d9b" alt="logo" />
        <div style="z-index: 10; display: flex; flex-direction: column; gap: 0.75rem;">
            <p style="font-family: Poppins, sans-serif;">Hi <strong>user@email.com</strong>,</p>
            <h1 style="font-family: Vidaloka,serif; font-size: 3rem; line-height: 1.375;">Welcome to The Subscription!</h1>
        </div>
    </section>
</div>
```

![image](https://user-images.githubusercontent.com/19299378/204335249-568d5a10-a066-4896-b8ea-d1b793a381d8.png)

What i get when i received the email:

![image](https://user-images.githubusercontent.com/19299378/204335410-6a9a2dde-9d69-4a83-ba85-2ec058ecb793.png)

Yes it got more broken

So I propose to you if we can get some Plain editor, so that we can send templated emails without the need to get the content parsed by the editor. The use case for this is quite big and impactful, like we don't need to set up our own nodemailer in our apps to send a templated email again.

I'm sorry if my English is hard to understand, it's not my native language, but i hope you got the point what i meant here, i hope to get response from you guys as soon as possible, thank you!

**Comment by azrikahar** [comment]
I'm by no means experienced with this so I could be wrong, but I think the more "resilient" approach would be to use something like https://github.com/maizzle/maizzle or https://github.com/soheilpro/mailwind (though big disclaimer that I've yet to use them myself) as I believe they'll convert the classes to inline styles in the build process rather than using a cdn script. Additionally CDN script is likely much larger than the classes that are in use, and I do wonder does it gets blocked in certain email clients.

There is also an open PR #15829 that will allow usage of email templates, which should also facilitate use case of precompiled email templates. 

That being said, this by no means invalidates the suggested "Plain" option, but more so to offer potential alternatives. 👍

**Reply by raflymln** [reply]
Thank you for the response! 😃👍

For the inline styles, I've try them, as you can see in number 3 above, but still the WYSIWYG editor messed it up so when it reached the email, the looks is not the same as when you see it on the editor.

Also love to hear that **theme** option is being worked on since that could be a better alternative for this

Let's see if the **theme** option can fix this problem or not, but if not, well i think the best way is to add **Plain** option then.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
